Tests/SwiftDocCUtilitiesTests/Test Bundles/TestBundle.docc/sidekit.symbols.json

```diff
@@ -77,7 +77,6 @@
         }
       ],
       "pathComponents" : [
-        "NonExistent",
```

Comment:
This path component is inaccurate. It doesn't reflect the symbol hierarchy and it isn't reflected on the `angle` property (which only has `["UncuratedClass", "angle"]` as its path components.